content/governance.md

```diff
@@ -32,7 +32,15 @@ Team member status may be given to those who have made ongoing contributions to
 
 New members may be proposed by any existing member by email to [prometheus-team][team]. It is highly desirable to reach consensus about acceptance of a new member. However, the proposal is ultimately voted on by a formal [supermajority vote](#supermajority-vote).
 
-Team members are added to the [GitHub organization][gh] as _Owner_. They should however respect the maintainers of each project.
+If the new member proposal is accepted, the proposed team member should be contacted privately to confirm or deny their acceptance of team membership. If they choose to accept, the following steps should be taken:
```

Comment:
"If they choose to accept, the following steps are taken:"

Reply:
SGTM.

Reply:
I would contact them CC'ed to the team list. Abuse is _very_ unlikely here, but there should be a paper trail on this.